tpp-app/.gitignore

```diff
@@ -53,3 +53,5 @@ buck-out/
 # Expo
 .expo/
 web-build/
+
+/src/onboarding/LoadingScreen.js
```

Comment:
why is this in the gitignore?

Reply:
LoadingScreen.js is the file for the splash / loading screen that I created when working on #20. At the time Chloe suggested that I don't link it up with anything yet and add it to .gitignore for now.

Reply:
cool & hip